ten-leetcode-problems/ContainsDoublicated.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ten_leetcode_problems
```

Comment:
we don't usually put underscores in the namespaces nor dashes in the project/folder names. Rename the folder to `TenLeetcodeProblems`, and update the namespace.